chat-plugins/scavengers.js

```diff
@@ -807,12 +823,13 @@ let commands = {
 	 */
 	createpractice: 'create',
 	createofficial: 'create',
+	createunrated: 'create',
```

Comment:
Is this for creating hunts that won't give out any points?

Reply:
yes, requested by a ro